README.md

```diff
@@ -1,12 +1,13 @@
 # Faraday
 
 Faraday is an HTTP client lib that provides a common interface over many
-adapters (such as Net::HTTP) and embraces the concept of Rack middleware when
+adapters (such as `Net::HTTP`) and embraces the concept of Rack middleware when
```

Comment:
Could we not wrap module names as inline code blocks? I am of the conviction that module names can occur in prose without marking them up as code. I think it looks better.